Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/Folders.cs

```diff
@@ -44,6 +51,59 @@ public static string GetFolderName(int folderArtifactId, IDBContext workspaceDbC
 			return GetFolderName(folderArtifactId, workspaceId, keplerHelper);
 		}
 
+		public static int GetRootFolderArtifactID(int workspaceID, IServicesMgr svgMgr, string userName, string password)
+		{
+			try
+			{
+				using (IRSAPIClient client = svgMgr.GetProxy<IRSAPIClient>(userName, password))
```

Comment:
Just noticed one part of this uses ConfigurationHelper for credentials and another uses params for credentials.  Should we make it consistent?

Reply:
No, changing that existing method is out of scope of this task.